pkg/network/events/monitor.go

```diff
@@ -136,10 +130,17 @@ func (h *eventConsumerWrapper) Copy(ev *model.Event) any {
 			if len(v) > 0 {
 				if t := envTagNames[k]; t != "" {
 					p.Tags = append(p.Tags, intern.GetByString(t+":"+v))
+					tagsFoundInEnvironment = true
 				}
 			}
 		}
 	}
+	if !tagsFoundInEnvironment {
```

Comment:
nit: you can instead just check if `len(p.Tags) == 0` here.

Reply:
I could, but I wanted to explicitly call it out.
1) environment variables supercede the various file parsing; my intent was to make the precedence more obvious
2) if someone adds some other way to get the tags, it becomes more obvious that the check for apmtags is related specifically to the environment block.

WDYT?

Reply:
Sounds good